app/src/main/res/values/styles_feedback.xml

```diff
@@ -3,6 +3,7 @@
     <style name="FeedbackTitle">
         <item name="android:textSize">@dimen/text</item>
```

Comment:
Hmm... I think so because this text has bold style. Choose which you prefer :+1: